scenarioo-client/app/build/use-cases-overview/use-cases-overview.component.html

```diff
@@ -107,7 +107,7 @@
                 </thead>
                 <tbody>
                 <tr (click)="goToUseCase(dynamicUseCase.name)"
-                    *ngFor="let dynamicUseCase of usecases | orderBy: order:reverse:true | filter: searchTerm
+                    *ngFor="let dynamicUseCase of usecases | orderBy: order:reverse:true | scFilterArray: searchTerm
```

Comment:
why dynamic?

Reply:
The usecases are dynamically loaded into the table? Which name do you suggest?

Reply:
I would suggest just to use `usecase` - it is obvious from context that this gets somehow loaded or injected dynamically, this does not have to be in the name otherwise we would have to name every thing `dynamicXy` that is dynamically loaded.